lib/schemas/service.js

```diff
@@ -34,14 +34,21 @@ const noAuth =
   Joi
     .boolean();
 
+const metadata =
+  Joi
+    .object()
+    .keys({
+      noAuth,
+      middleware,
+    });
```

Comment:
@CodeFred - Thoughts on keeping a whitelist of allowed properties? Otherwise we lose the validation